src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -58,12 +58,12 @@
     <XunitArguments>$(TargetFileName) $(XunitOptions)</XunitArguments>
 
     <TestProgram Condition="'$(TestHost)'!=''">$(TestHost)</TestProgram>
-    <TestArguments Condition="'$(TestHost)'!=''">$(XunitExecutable) $(XunitArguments)</TestArguments>
+    <TestArguments Condition="'$(TestHost)'!=''">$(XunitExecutable) $(XunitArguments) {XunitTraitOptions}</TestArguments>
 
     <TestProgram Condition="'$(TestHost)'==''">$(XunitExecutable)</TestProgram>
-    <TestArguments Condition="'$(TestHost)'==''">$(XunitArguments)</TestArguments>
+    <TestArguments Condition="'$(TestHost)'==''">$(XunitArguments) {XunitTraitOptions}</TestArguments>
```

Comment:
So does that mean when run through VS the xunit traits aren't getting added (because they're only added in that target).

The non-placeholder version doesn't work for code coverage, so we'll need to come up with something else, like a postfix